src/exchanges/__snapshots__/fetch.test.ts.snap

```diff
@@ -144,7 +144,7 @@ Object {
         ],
         "results": Array [
           Object {
-            "isThrow": false,
+            "type": "return",
```

Comment:
@kitten is this change in return type expected?

Reply:
Yep, seems to be due to the Jest upgrade